build/RELEASING.md

```diff
@@ -1,9 +1,34 @@
 # Release process
 
-Only for Maintainers.
+**Only for Maintainers.**
 
- 1. Decide the tag name (version name) to be released. e.g. 1.4.0-beta.1,
-    1.0.0-rc9.7 etc.
+ 1. Decide the component(s) and tag name (version name) to be released.
+
+    Notes:
+
+       * There are different categories of packages. Check the project file for
+         what you want to release and look for `MinVerTagPrefix`.
+
+         * `core-`: Core packages. These packages are defined\goverened by the
+           OpenTelemetry Specification and have released stable versions. They
+           may be released as `alpha`, `beta`, `rc`, or stable.
+
+         * `coreunstable-`: Core unstable packages. These packages are
+           defined\goverened by the OpenTelemetry Specification but have not
+           released stable versions. They may be released as `alpha` or `beta`.
+
+         * Everything else: Instrumentation packages have dedicated tags. Some
+           packages have released stable and some have not. These packages may
+           be released as `alpha`, `beta`, `rc`, or stable depending on the
+           stability of the semantic conventions used by the instrumentation.
+
+       * Instrumentation packages and core unstable packages always depend on
+       the stable versions of core packages. Before releasing a non-core
+       component ensure the `OTelLatestStableVer` property in
+       `Directory.Packages.props` has been updated to the latest stable core
+       version.
+
+       * Core unstable packages may only be released as `alpha` or `beta`.
```

Comment:
How to interpret this? Will the RC versions be deprecated/unlisted/removed?

https://www.nuget.org/packages/OpenTelemetry.Exporter.Prometheus.AspNetCore/1.8.0-rc.1#versions-body-tab

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/17327289/d77f0e07-1f69-466e-a994-56996788d429)

Reply:
Sounds good.